antsibull/cli/doc_commands/devel.py

```diff
@@ -2,7 +2,140 @@
 # License: GPLv3+
 # Copyright: Ansible Project, 2020
 
+import asyncio
+import os
+import os.path
```

Comment:
This isn't really necessary
```suggestion
```

Reply:
importing os.path when it is used is best practice.  The fact that merely importing os allows os.ath to work is an implementation detail of how the os module is implemented.

Reply:
This is probably subjective — some of my linters think that the best practice is not to do this :man_shrugging:

Reply:
That sounds like a bug/misfeature in the linters.

This behaviour of os and os.path has confused many people about python import semantics over the years (ask jimi-c about it sometime ;-).